bounties_api/analytics/management/commands/timeline_generator.py

```diff
@@ -260,10 +267,41 @@ def handle(self, *args, **options):
                 # and provides a better way to expose by day or by hour in case of been needed
                 for day in days:
                     bounty_day = generate_timeline(day_bounds(day), schema=schema)
-                    bounty_points = BountiesTimeline.objects.filter(date=day.date(), schema=schema)
+                    bounty_points = BountiesTimeline.objects.filter(date=day.date(), schema=schema, is_week=False)
 
                     if bounty_points.exists():
                         bounty_point = bounty_points.first()
                         bounty_day.id = bounty_point.id
 
                     bounty_day.save()
+
+        for schema in schemas:
+            needs_genesis = not BountiesTimeline.objects.filter(schema=schema, is_week=True).exists()
+
+            first_date = BountyState.objects.first()
+            last_date = datetime.utcnow()
+
+            if needs_genesis:
+                bounties_by_week = range_weeks(first_date.change_date, last_date.change_date + timedelta(days=7))
```

Comment:
@villanuevawill 
This piece of code is responsible that the current week is also included in the calculations.
So as of now, it works as you described.